sdk/quota/arm-quota/src/api/quotaOperation/operations.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import type { AzureQuotaExtensionAPIContext as Client } from "../index.js";
+import type { _OperationList, OperationResponse } from "../../models/models.js";
+import { _operationListDeserializer, exceptionResponseDeserializer } from "../../models/models.js";
+import type { PagedAsyncIterableIterator } from "../../static-helpers/pagingHelpers.js";
+import { buildPagedAsyncIterator } from "../../static-helpers/pagingHelpers.js";
+import { expandUrlTemplate } from "../../static-helpers/urlTemplate.js";
+import type { QuotaOperationListOptionalParams } from "./options.js";
+import type { StreamableMethod, PathUncheckedResponse } from "@azure-rest/core-client";
+import { createRestError, operationOptionsToRequestParameters } from "@azure-rest/core-client";
+
+export function _listSend(
+  context: Client,
+  options: QuotaOperationListOptionalParams = { requestOptions: {} },
+): StreamableMethod {
+  const path = expandUrlTemplate(
+    "/providers/Microsoft.Quota/operations{?api%2Dversion}",
```

Comment:
could you add a test case for this? this is a tenant-level operation without subscriptionId.